src/Commands/MigrateFresh.php

```diff
@@ -19,7 +19,10 @@ class MigrateFresh extends Command
      *
      * @var string
      */
-    protected $signature = 'migrate:fresh {--seed} {--force}';
+    protected $signature = 'migrate:fresh {--database= : The database connection to use.}
+                {--force : Force the operation to run when in production.}
+                {--path= : The path of migrations files to be executed.}
```

Comment:
From the docs: "If the user must specify a value for an option, suffix the option name with a = sign"

So it makes sense here!